src/__template__/ComponentTemplate.vue

```diff
@@ -90,6 +100,76 @@ const rootElementStyles = computed((): Record<string, string> => ({
   lineHeight: '0',
   width: iconSize.value,
 }))
+
+/**
+ * Prefixes all IDs within the <defs> section of an SVG element and updates all references to these IDs. This is useful to avoid ID conflicts when embedding multiple SVGs in a document.
+ *
+ * The function performs the following steps:
+ * 1. Finds all direct children of the <defs> element that have an `id` attribute and prefixes their IDs with a random string.
+ * 2. Updates all references to these prefixed IDs in attributes like `url(#id)`, `href`, `xlink:href`, etc.
```

Comment:
```suggestion
 * Prefixes all ids within the SVG element and update all corresponding references to these ids. This is useful to avoid id conflicts when utilizing multiple instances of the same SVG in the DOM.
```